op-node/flags/flags.go

```diff
@@ -100,6 +100,12 @@ var (
 		Required: false,
 		Hidden:   true,
 	}
+	L1RPCMaxConcurrentRequests = &cli.IntFlag{
+		Name:    "l1.max-concurrent-rpc-requests",
+		Usage:   "Maximum number of concurrent RPC requests to make to the L1 RPC provider.",
+		EnvVars: prefixEnvVars("L1_MAX_CONCURRENT_RPC_REQUESTS"),
+		Value:   20,
```

Comment:
```suggestion
		Value:   10,
```

> It defaults to 20, the same value as in the default eth config. 

It actually defaults to 10 in the config. The max-requests-per-batch is 20 (and already configurable)

https://github.com/ethereum-optimism/optimism/blob/301e996d776a4c76e809861db96f000f89069bb6/op-service/sources/l1_client.go#L40

Increasing it to 20 might cause some users to hit RPC rate-limits on the server side that they may not have before, but probably harmless.